apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationPreviewList.tsx

```diff
@@ -0,0 +1,186 @@
+  fragment ReviewApplicationPreviewList on PoolCandidate {
+    ...ReviewApplicationDialog
+    id
+    finalDecisionAt
+    submittedAt
+    removedAt
+    finalDecision {
+      value
+    }
+    assessmentStatus {
+      assessmentStepStatuses {
+        step
+      }
+      overallAssessmentStatus
+      currentStep
+    }
+    pool {
+      id
+      name {
+        localized
+      }
+      classification {
+        group
+        level
+        minSalary
+        maxSalary
+      }
+      closingDate
+      areaOfSelection {
+        value
+      }
+      screeningQuestions {
+        id
+      }
+    }
+  }
+`);
+
+interface ReviewApplicationPreviewListProps {
+  applicationsQuery: FragmentType<
+    typeof ReviewApplicationPreviewList_Fragment
+  >[];
+}
+
+const ReviewApplicationPreviewList = ({
+  applicationsQuery,
+}: ReviewApplicationPreviewListProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  const applications = getFragment(
+    ReviewApplicationPreviewList_Fragment,
+    applicationsQuery,
+  );
+
+  return (
+    <>
+      {applications.length ? (
+        <PreviewList.Root>
+          {applications.map((application) => {
+            const { id, pool, submittedAt, finalDecisionAt } = application;
+
+            const status = getApplicationStatusChip(
+              application.submittedAt,
+              pool.closingDate,
+              application.removedAt,
+              application.finalDecisionAt,
+              application.finalDecision?.value,
+              pool.areaOfSelection?.value,
+              application.assessmentStatus,
+              pool.screeningQuestions,
+              intl,
+            );
+
+            const applicationMetadata: PreviewMetaData[] = [
+              {
+                key: "status",
+                type: "chip",
+                color: status.color,
+                children: status.label,
+              },
+              {
+                key: "classification",
+                type: "text",
+                children: pool?.classification
+                  ? getClassificationName(pool?.classification, intl)
+                  : intl.formatMessage(commonMessages.notFound),
+              },
+              {
+                key: "date",
+                type: "text",
+                children: (
+                  <ApplicationDate
+                    closingDate={pool?.closingDate}
+                    submittedAt={submittedAt}
+                    finalDecisionAt={finalDecisionAt}
+                    status={status.value}
+                  />
+                ),
+              },
+            ];
+
+            return (
+              <PreviewList.Item
+                key={id}
+                title={
+                  pool.name?.localized
+                    ? intl.formatMessage(
+                        {
+                          defaultMessage:
+                            "<hidden>Application for </hidden>{poolName}",
```

Comment:
I don't think you can fix this and I don't expect you to even attempt it here since it is way out of scope. But I just need to highlight that our hidden text is cuaisng issues in both voiceover and NVDA.

This reads in NVDA as such:

 - Heading level 4, application for
 - Heading level 4, {poolName}
 
 This is incredibly confusing since it seems as though there are two separate headings here when there are not. I feel like we need to reduce our usage of hidden text where possible if this is going to continute to be an issue in 2/4 most commonly used screen readers 👎

Reply:
I'll bring this up in retro again.